### PR TITLE
Feature/as 556602591218327 add styling for header

### PR DESCRIPTION
**Pull request by joshJarr**
https://app.asana.com/0/2524932960023

### CODE REVIEW

File: bakerydemo/templates/includes/header.html [hunk 1]
```diff
@@ -3,24 +3,27 @@
...
+                </a>
 
-                <form action="/search" method="get" class="search" _lpchecked="1" role="search">
+                <!-- <form action="/search" method="get" class="search" _lpchecked="1" role="search">
                     <input name="q" type="text" placeholder="Search the site" id="" aria-label="Search the site" autocomplete="off">
                     <a href="#" class="search-icon">
                         <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 451 451"><path d="M447.05 428l-109.6-109.6c29.4-33.8 47.2-77.9 47.2-126.1C384.65 86.2 298.35 0 192.35 0 86.25 0 .05 86.3.05 192.3s86.3 192.3 192.3 192.3c48.2 0 92.3-17.8 126.1-47.2L428.05 447c2.6 2.6 6.1 4 9.5 4s6.9-1.3 9.5-4c5.2-5.2 5.2-13.8 0-19zM26.95 192.3c0-91.2 74.2-165.3 165.3-165.3 91.2 0 165.3 74.2 165.3 165.3s-74.1 165.4-165.3 165.4c-91.1 0-165.3-74.2-165.3-165.4z"></path></svg>
                     </a>
-                </form>
+                </form> -->
```
Comment (joshJarr) [inline bakerydemo/templates/includes/header.html:17]: unsure if this search is required. Commented out for now.

File: readme.md [hunk 2]
```diff
@@ -63,10 +63,11 @@ Run the following commands:
 git clone git@github.com:wagtail/bakerydemo.git
 cd bakerydemo
 docker-compose up --build -d
+docker-compose run app /venv/bin/python manage.py createsuperuser #Follow the instructions, name your user `admin`
```
Comment (joshJarr) [inline readme.md]: Lemme know if this seems vague, or is even necessary.
Needed to run this before I could load the test data.

File: readme.md [hunk 3]
```diff
@@ -63,10 +63,11 @@ Run the following commands:
 git clone git@github.com:wagtail/bakerydemo.git
```
Comment (gerbyzation) [inline readme.md]: This is the wrong URL, and HTTPS might be preferable as a default?

**Review by gerbyzation — CHANGES_REQUESTED** [review]

**Review by thisisthechris — APPROVED** [review]
Looks great, can't spot any issues.

**Pull request merged**